<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QDialog" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SplashScreen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 249, 247);
background-color: #4C566A;
border-radius:10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-3</x>
        <y>50</y>
        <width>661</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>中国&lt;strong&gt;象棋&lt;/strong&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;选择模式：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>661</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;loading..&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>641</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Created: Luke&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>461</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="mode1">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #E5E9F0;
background-color: #5E81AC;
border: none;
border-radius:7px;</string>
         </property>
         <property name="text">
          <string>与自己对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mode2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #E5E9F0;
background-color: #5E81AC;
border: none;
border-radius:7px;</string>
         </property>
         <property name="text">
          <string>与AI对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mode3">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #E5E9F0;
background-color: #5E81AC;
border: none;
border-radius:7px;</string>
         </property>
         <property name="text">
          <string>网络对战</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>531</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
background-color: #434C5E;
color: #D8DEE9;
border-style: none;
border-radius:10px;
text-align:center;
}
QProgressBar::chunk {
border-radius:10px;
background-color: qlineargradient(spread:pad, x1:0, y1:0.369, x2:1, y2:0.727, stop:0 rgba(0, 135, 202, 255), stop:1 rgba(67, 92, 255, 255));
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
